src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProvider.cs

```diff
@@ -113,6 +113,17 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
             }
         }
 
+        private bool IsPropertyGetOrSetMethod(ISymbol symbol)
+        {
+            if (symbol.Kind == SymbolKind.Method)
+            {
+                var methodSymbol = symbol as IMethodSymbol;
```

Comment:
nit: consider using pattern matching

Reply:
Good suggestion, I switched to use pattern matching.

Reply:
I think just doing `if (symbol is IMethodSymbol methodSymbol)` is a bit more idiomatic here. If nothing else, there's no reason for the as: we know that cast shouldn't fail and so a direct `(IMethodSymbol)symbol)` cast is better. If that fails, we'll know we have problems!

Reply:
Yes, that's what I had in mind when I said pattern matching - just `if (symbol is IMethodSymbol method)`

Reply:
That makes sense. I will keep this advice in mind for the future. Thanks for the advice!